### DISCUSSION TITLE
how to center a title (h1,h2,h3)

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#1178] [Q&A] [answered]
just like:
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/16400882/192406612-e68e3ae6-ebca-4cd2-b6dc-716e6ce4ba70.png">

**Comment by JessRynkar** [comment, marked as answer]
Hey @zhounewz - this feature is on our roadmap [here](https://github.com/payloadcms/payload/discussions/1218), for now you can achieve this by adding a custom element. Read the comments [here for more info](https://github.com/payloadcms/payload/discussions/1218).
